package.json

```diff
@@ -30,6 +30,6 @@
     "grunt-release": "latest"
   },
   "engines": {
-    "node": ">=0.10"
+    "node": "6.11.1"
```

Comment:
Same comment as in Hilary - we want later Node versions to continue working so needs a `>=`.